kafka-rest/src/main/java/io/confluent/kafkarest/resources/v2/PartitionsResource.java

```diff
@@ -200,13 +197,56 @@ public void onCompletion(
     );
   }
 
-  private boolean topicExists(final String topic)  throws Exception {
-    return ctx.getAdminClientWrapper().topicExists(topic);
+  /**
+   * Get offsets for the given {@code topic} and {@code partition}.
+   *
+   * <p>If {@code timestamp} is not specified, returns a summary with beginning and end offsets.
```

Comment:
out of date comment - timestamp can no longer be specified.

Reply:
Done